k8s/deployment.yaml

```diff
@@ -0,0 +1,20 @@
+# deployment.yaml
```

Comment:
Fine if you're interested, but there's no need to keep around this nginx deployment once you've got the todo app working